extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-build-steps.js

```diff
@@ -1,29 +1,135 @@
-import { LitElement, html, css} from 'lit';
-import { buildSteps } from 'devui-data';
+import { LitElement, html, css } from 'lit';
+
+import { JsonRpc } from 'jsonrpc';
+import { until } from 'lit/directives/until.js';
+import '@vaadin/grid';
+import { columnBodyRenderer } from '@vaadin/grid/lit.js';
+import '@vaadin/grid/vaadin-grid-sort-column.js';
+import '@vaadin/icon';
+import '@vaadin/text-field';
+import '@vaadin/vertical-layout';
+import '@vaadin/horizontal-layout';
 
 /**
  * This component shows the Build Steps
  */
 export class QwcBuildSteps extends LitElement {
+
+  jsonRpc = new JsonRpc(this, true, 'build-metrics');
```

Comment:
Omit the name here so:
```suggestion
  jsonRpc = new JsonRpc(this, true);
```